### PR TITLE
Bugfix/3924 notification created in scheduler run

### LINKED ISSUE #3924: Notification created of each process can't be deactivate
There is a Notification Queue created by each Process of Flush System Queue that is generated, and this notification try to send an emaill with the LOG of the Process.

image

image

In our case, there is no account name defined in the AD_NotificationRecipient

image

And in the scheduler Recipient we have it empty

image

https://user-images.githubusercontent.com/31936815/174117791-d200ef28-319c-439d-8a1b-bb90034be77c.mp4

In order of this, i understand that the bug is that the process for some reason is creating a Notification System and a Notification Queue after every process and it shouldnt be like this. 

There must be some way to define if you want to create this notification or not. If you want, i understand that it would be enough to define the Scheduler Recipient to recieve it, so i dont see the necesity to create an other notification every time the process run without the chance of define the User (it is created to de SuperUser and I don't know where it is defined).
 

Unless you define a User in the "Scheduler Recipient" AD_SchedulerRecipient WHERE AD_User_ID?

### PR DESCRIPTION
**Pull request by rcapecce**
Fix #3924

### CODE REVIEW

**Comment by yamelsenih** [comment]
Hello @rcapecce maybe you can resolve this build error: https://github.com/adempiere/adempiere/actions/runs/6935572892/job/18865975321#step:4:37

**Comment by rcapecce** [comment]
> Hello @rcapecce maybe you can resolve this build error: https://github.com/adempiere/adempiere/actions/runs/6935572892/job/18865975321#step:4:37

Sure, this was solved @yamelsenih

File: base/src/org/compiere/model/MProduction.java [hunk 1]
```diff
@@ -44,7 +44,6 @@
 import org.compiere.util.Env;
 import org.compiere.util.Msg;
 import org.compiere.util.Util;
-import org.eevolution.manufacturing.model.MPPProductBOMLine;
```
Comment (yamelsenih) [inline base/src/org/compiere/model/MProduction.java:47]: Please, this pull request should be atomic. A production class is unnecessary for this scope.

Can you use a fresh branch?

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: base/src/org/compiere/model/MProduction.java (hunk 1 above)
Comment (rcapecce) [reply base/src/org/compiere/model/MProduction.java]: Sure, I checkout master and pick this commits again without this change

**Review by yamelsenih — APPROVED** [review]

**Pull request merged**